<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="productDisplay">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>461</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Последне обновление: никогда</string>
    </property>
   </widget>
   <widget class="QToolButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>550</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon-refresh-128.png</normaloff>icon-refresh-128.png</iconset>
    </property>
   </widget>
   <widget class="QTreeWidget" name="catalogTreeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>481</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Каталог продукции</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Продуктов:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>271</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Категорий</string>
    </property>
   </widget>
   <widget class="QLabel" name="categoryCounter">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>500</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="productCounter">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>520</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
